src/engine/bufferscalers/enginebufferscalerubberband.cpp

```diff
@@ -404,7 +413,10 @@ void EngineBufferScaleRubberBand::reset() {
     }
     while (remaining_padding > 0) {
         const size_t pad_samples = std::min<size_t>(remaining_padding, block_size);
-        m_pRubberBand->process(m_bufferPtrs.data(), pad_samples, false);
+        {
+            ScopedTimer t(u"RubberBand::process");
```

Comment:
also in light of #13258, use `QStringLiteral`

Reply:
> scoped timer is not very realtime save afai can tell.

Does that matter? I think this is mostly to give a broad overview of the where most of the CPU time goes. What else should I use otherwise?

Reply:
> Does that matter? 

It does in realtime code where non-realtime-safe operations (such as (de-)allocating memory) can cause the audio code to run for too long (resulting in buffer underruns). In #13236 I tried to reduce that a little, but the underlying `StatManager` member functions are not realtime safe (in fact they lock, though I don't know how contended that lock is). 

> What else should I use otherwise?

I don't really know. I think the other solutions we had were fairly ugly/low level, so there is no good way of tracking audio code right now. I think you can keep it, I'm just saying that it could result in buffer-underruns with `--developer`. Since its limited to `--developer`, its probably okay. In the longterm, we should consider refactoring StatsManager to use a lock-free queue instead.

Reply:
Ah good point. 
Do you think it's best to remove entirely or since it limited to `--developer`, it is fine to keep?

Reply:
I'd say its fine to keep. If you really can't tolerate underruns in `--developer` you can max out the buffersize, that should probably work around that. Though in the mid-term future, we should get `StatsManager` lock-free (ideally wait-free too).

Reply:
but I'd like to get @daschuer's opinion on this.

Reply:
We use the ScopedTimer in other places of the engine already. So it is fine here as well IMHO.